packages/transformers/markdown/src/MarkedTransformer.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+import {promisify} from '@parcel/utils';
+import marked from 'marked';
```

Comment:
We might want to look into using something like [remark](https://github.com/remarkjs/remark) for markdown since it supports all kinds of plugins for different things. For example, syntax highlighting could be done by a plugin for remark however people like without needing Parcel to build it into the transformer. Thoughts?

Reply:
Not sure, wouldn't that require a javasrcript based config?